src/views/mypage/components/report/components/AnalysisCard.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="bg-ivory rounded-lg p-6 w-full h-full">
+    <div class="rounded-lg flex flex-col gap-y-1">
+      <!-- 제목 -->
+      <span class="text-gray-600 text-[14px]">
+        {{ title }}
+      </span>
+
+      <!-- 차트 모드 -->
+      <div v-if="mode === 'chart'" class="flex items-center justify-between">
+        <!-- 원형 차트 -->
+        <div class="relative w-22 h-22">
+          <svg class="w-22 h-22 transform -rotate-90" viewBox="0 0 120 120">
+            <!-- 배경 원 -->
+            <circle
+              cx="60"
+              cy="60"
+              r="50"
+              fill="none"
+              stroke="#e5e7eb"
+              stroke-width="10"
+            />
+            <!-- 진행률 원 -->
+            <circle
+              cx="60"
+              cy="60"
+              r="50"
+              fill="none"
+              stroke="#228b22"
+              stroke-width="10"
+              stroke-dasharray="314.16"
+              :stroke-dashoffset="
+                314.16 - (314.16 * chartData.percentage) / 100
+              "
+              stroke-linecap="round"
+            />
+          </svg>
+          <div class="absolute inset-0 flex items-center justify-center">
+            <span class="text-sm text-green">{{ chartData.percentage }}%</span>
+          </div>
+        </div>
+
+        <!-- 카테고리별 데이터 -->
+        <div class="flex flex-col space-y-2">
+          <div
+            v-for="category in chartData.categories"
```

Comment:
```
const { percentage, categories } = props.chartData;
```
charData.categories가 여러 번 쓰인다면 이런 식으로 선언해놓고

```suggestion
            v-for="category in categories"
```
이렇게 써도 좋을 것 같습니당

Reply:
감사합니다! 가독성 측면에서 좋은 것 같다고 생각이 들어서 charcterData도 함께 수정했습니다!
[1590889](https://github.com/kb-jjukkumi/choogoomoneyna_FE/pull/48/commits/1590889c86b929d4ded091a156bc015047a28d9d)

Reply:
제가 작업을 헷갈려서 마이페이지가 아닌 컴포넌트에도 반영을 했습니다! 나중에 이 두 컴포넌트를 합칠 수 있는지 고민해보겠습니다!

마이페이지- 자산 분석 페이지 수정 [ffd1077](https://github.com/kb-jjukkumi/choogoomoneyna_FE/pull/48/commits/ffd10773c9424439a6e9de77a55dd8c3347640e8)

첫 로그인 때 보여주는 자산 분석 수정 [1590889](https://github.com/kb-jjukkumi/choogoomoneyna_FE/pull/48/commits/1590889c86b929d4ded091a156bc015047a28d9d), [9b44de7](https://github.com/kb-jjukkumi/choogoomoneyna_FE/pull/48/commits/9b44de7d955e645a820b9278dfea2c6213987a86)